docs/get-started/roles-and-permissions/lbac.md

```diff
@@ -22,20 +22,60 @@ PMM supports standard as well as custom labels. PMM automatically assigns standa
 
 ## Adding labels
 
-You can add custom or standard labels in PMM while adding a service for monitoring in PMM. 
+In PMM, labels enable you to identify, categorize, and organize your monitored services and resources. Labels also simplify filtering, grouping, and analyzing data. 
+
+While adding a service for monitoring in PMM, you can add custom or standard labels. The following labels are available.
```

Comment:
WRT `The following labels are available.`: it seems like it was meant to list all the possible labels, right? If not, let's remove it.

Reply:
yes.

Reply:
Removed.